src/components/Tooltip/internal/Tooltip.module.scss

```
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use '../../../style/variables/colors.scss';
+@use '../../../style/variables/spaces.scss';
+@use '../../../style/mixins/font-config.scss';
+
+.tooltip {
+  color: map.get(colors.$colors, 'base-white');
+  position: relative;
+  &:hover span {
+    visibility: visible;
+  }
+  span {
+    visibility: hidden;
```

On the tooltip we can add an animation so the text appearing is smoother

```suggestion
  &:hover span {
    transition: visibility 150ms ease, opacity 150ms ease;
    visibility: visible;
    opacity: 1;
  }
  span {
    transition: visibility 150ms ease, opacity 150ms ease;
    visibility: hidden;
    opacity: 0;
```

We cannot animate `visibility` alone so adding an opacity will help us do it

---

It's better indeed!